src/lib.ts

```diff
@@ -30,10 +29,6 @@ export default async function run(
       ...configuration
     }
 
-    // Defines the repository paths and token types.
-    settings.repositoryPath = generateRepositoryPath(settings)
-    settings.tokenType = generateTokenType(settings)
```

Comment:
Is there a purpose for moving these over to the git init function instead of leaving it here? `run` is the initiator so I think they make more sense here.

Reply:
I get your point. So this way I'm not sure where `checkParameters` should be.

Reply:
I think here would make sense right?

Reply:
Yes. I need to refactor some tests too.